pages/rooms/[roomId].tsx

```diff
@@ -8,30 +8,15 @@ const RoomId = () => {
     roomId: string;
     client_token: string;
     refresh_token: string;
-    network_metrics: string;
-    simulcast: string;
-    dial_out: string;
-    use_mixed_audio: string;
-  };
-
-  const optionalFeatures = {
-    isNetworkMetricsEnabled: queryParameters.network_metrics === 'true',
-    isSimulcastEnabled: queryParameters.simulcast === 'true',
-    isDialOutEnabled: queryParameters.dial_out === 'true',
-    useMixedAudioForOutput:
-      queryParameters.use_mixed_audio === 'false' ? false : true, // by default this is true
   };
 
   return (
     <>
-      {router.isReady && (
-        <Rooms
```

Comment:
The only component/side-effect that has to wait for the router to be ready, is the side-effect where we `getUserMedia` right after media preview mounts, and video is enabled.

`router.isReady` has been moved to that component.